### ISSUE TITLE
Script only deploys tag & nothing else to WP SVN 

### ISSUE THREAD

**Issue by CodyFitzpatrick** [GaryJones/wordpress-plugin-svn-deploy#33]
Came across this script after researching more automated methods of deploying to SVN.

After cloning it to the parent folder of my WP plugin git repo, and running the script, it appears to have deployed only the tag name to SVN.

You can see here (until I re-deploy), that only the tag name "1.0.0" was pushed: http://plugins.svn.wordpress.org/visual-feedback/

All looked good but then I started to get some errors. Here is part of the log, when the errors started occurring:

```
OK to proceed (Y|n)? y

..........................................

Preparing to deploy WordPress plugin

..........................................


Changing to /Users/ME/FOLDER_1_NAME/FOLDER_2_NAME/FOLDER_3_NAME/visual-feedback
Git tag 1.0.0 does exist. Let's continue...

Creating local copy of SVN repo trunk...
A    /tmp/visual-feedback/assets
A    /tmp/visual-feedback/branches
A    /tmp/visual-feedback/tags
A    /tmp/visual-feedback/trunk
Checked out revision 1875910.
Ignoring GitHub specific files
property 'svn:ignore' set on '/tmp/visual-feedback/trunk'
Exporting the HEAD of master from git to the trunk of SVN

Moving assets.
mv: rename /tmp/visual-feedback/trunk/assets/* to /tmp/visual-feedback/assets/*: No such file or directory
svn: E125001: '/tmp/visual-feedback/trunk/assets' does not exist

Changing directory to SVN and committing to trunk.
xargs: illegal option -- r
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
xargs: illegal option -- r
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
Sending        .
Authentication realm: <https://plugins.svn.wordpress.org:443> Use your WordPress.org login
Password for 'MYUSERNAME': **************MYPASSWORD**************

Committing transaction...
Committed revision 1875913.

Updating WordPress plugin repo assets and committing.
xargs: illegal option -- r
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
xargs: illegal option -- r
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]

Creating new SVN tag and committing it.
svn: E125001: '/tmp/visual-feedback/tags/1.0.0/assets' does not exist
svn: E125001: '/tmp/visual-feedback/tags/1.0.0/trunk' does not exist
Adding         .
Committing transaction...
Committed revision 1875915.

Removing temporary directory /tmp/visual-feedback.
*** FIN ***
```

It appears the script is not properly copying over the files of the plugin's parent directory. My first thought, as I am new to this script, is it requires you to set up the necessary SVN folders (assets, branches, tags, & trunk) prior to running it, but found that is not the case after further reading, hence one of the benefits of using this script.

Any ideas? Is it due to xargs on MacOS not supporting the --replace option?

**Comment by GaryJones** [comment]
Thanks for the report @CodyFitzpatrick.

Obviously, testing a fresh plugin is harder to check these things. It's been a while since I did that, so it may be an incorrect assumption in the script that `/trunk`, `/assets`, `/tags` and `/branches` exist in a _fresh_ SVN repo on wordpress.org. Explicit automated creation of these directories may be needed for the first time if the initial checkout doesn't result in those directories appearing in `/tmp/your-plugin/*`. Are you able to confirm if fresh SVN repos on wordpress.org have these directories?

----

Alternatively, if the directories are created, but are not added as part of the `svn add stage, if you manually undo https://github.com/GaryJones/wordpress-plugin-svn-deploy/commit/b2fb8d3d64e5f944eb06c3082d901f44a860df29 in your local copy of this script (or change to `-R`), and try to deploy again, does it work?

**Comment by CodyFitzpatrick** [comment]
Hey @GaryJones,

I just pushed an update to the plugin via the deploy.sh script with the following modification (same as my previous comment) and all went well:
- replaced all instances of `xargs -r` with `xargs`

I have not yet tried your recommendation as I figured it would be best tested with the push of a fresh plugin and I am only working on one plugin at this point in time. When/if I end up testing the swapping of `xargs -r svn del` for `xargs svn del` and likewise for svn add, I will certainly come back and here and share the results.

Thanks again for providing and maintaining this script.
